### PR TITLE
Update DCS doc

### PR DESCRIPTION
**Pull request by smilemusic**
$ make testacc TEST=./huaweicloud/ TESTARGS='-run TestAccDcsInstancesV1_basic -mod=readonly'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/ -v -run TestAccDcsInstancesV1_basic -mod=readonly -timeout 360m
=== RUN   TestAccDcsInstancesV1_basic
--- PASS: TestAccDcsInstancesV1_basic (357.85s)
PASS
ok      github.com/terraform-providers/terraform-provider-huaweicloud/huaweicloud       358.146s

### CODE REVIEW

File: website/docs/r/dcs_instance_v1.html.markdown [hunk 1]
```diff
@@ -16,89 +16,97 @@ Manages a DCS instance in the huaweicloud DCS Service.
...
-  password          = "Huawei_test"
-  engine            = "Redis"
-  capacity          = 2
-  vpc_id            = "1477393a-29c9-4de5-843f-18ef51257c7e"
+  name  = "%s"
+  engine_version = "3.0"
+  password = "Huawei_test"
+  engine = "Redis"
+  capacity = 2
+  vpc_id = "%s"
```
Comment (ShiChangkuo) [inline website/docs/r/dcs_instance_v1.html.markdown]: please reference: https://www.terraform.io/docs/configuration/style.html

**Comment by smilemusic** [comment]
In order to improve the user experience, the ProductID that created the dcs instance already supports id and name, and the doc needs to be updated

File: huaweicloud/resource_huaweicloud_dcs_instance_v1.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ func resourceDcsInstanceV1() *schema.Resource {
 			},
 			"engine_version": {
 				Type:     schema.TypeString,
-				Optional: true,
+				Required: true,
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_dcs_instance_v1.go]: `engine_version`  is required when `engine` is Redis, so just keep it optional.

File: website/docs/r/dcs_instance_v1.html.markdown [hunk 3]
```diff
@@ -21,84 +21,92 @@ resource "huaweicloud_networking_secgroup_v2" "secgroup_1" {
 }
 data "huaweicloud_dcs_az_v1" "az_1" {
   port = "8002"
+  code = "dcs.master_standby"
```
Comment (ShiChangkuo) [inline website/docs/r/dcs_instance_v1.html.markdown]: the availability zone is not exist?

**Pull request merged**